examples/http_download.json

```diff
@@ -4,12 +4,22 @@
 		{
 			"id": "source_path",
 			"type" : "string",
-			"value" : "https://media-io.com"
+			"value" : "index.html"
+		},
+		{
+			"id": "source_secret",
+			"type" : "truc",
+			"value" : {
+				"type": "http",
+				"endpoint": "http://localhost:8080",
+				"method": "GET",
+				"headers": "{\"content-type\": \"text/html\"}"
```

Comment:
the header don't contain that field.
Possible but not required is `accept` [doc](https://developer.mozilla.org/fr/docs/Web/HTTP/Headers/Accept).